README.md

```diff
@@ -1,2 +1,17 @@
 # gtsam-project-python
 Project template using GTSAM + python wrapping
+
+
+INSTALL
+=======
+- Python3 is required
```

Comment:
Python3 isn't explicitly required since the wrapper works for 2.7 as well, but since we are 6 months from 2.7 EOL, **3 is strongly recommended**.